Xamarin.Forms.Core.UnitTests/PickerTests.cs

```diff
@@ -1,59 +1,341 @@
 using System;
 
 using NUnit.Framework;
-using System.Collections.Generic;
+using System.Collections.ObjectModel;
 
 namespace Xamarin.Forms.Core.UnitTests
 {
-	[TestFixture]
-	public class PickerTests : BaseTestFixture
-	{
-		[Test]
-		public void TestSetSelectedIndexOnNullRows()
-		{
-			var picker = new Picker ();
+    internal class ContextFixture
+    {
+        public string DisplayName { get; set; }
```

Comment:
What I believe @StephaneDelcroix may have been referring to are the tabs, which should be hard tabs.